Foundation/URLCache.swift

```diff
@@ -40,6 +47,38 @@ extension URLCache {
     }
 }
 
+class StoredCachedURLResponse: NSObject, NSSecureCoding {
+    class var supportsSecureCoding: Bool { return true }
+    
+    func encode(with aCoder: NSCoder) {
+        aCoder.encode(cachedURLResponse.response, forKey: "response")
+        aCoder.encode(cachedURLResponse.data as NSData, forKey: "data")
+        aCoder.encode(cachedURLResponse.storagePolicy.rawValue, forKey: "storagePolicy")
+        aCoder.encode(cachedURLResponse.userInfo as NSDictionary?, forKey: "userInfo")
+        aCoder.encode(cachedURLResponse.date as NSDate, forKey: "date")
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        guard let response = aDecoder.decodeObject(of: URLResponse.self, forKey: "response"),
```

Comment:
A variant I have re indentation style is that the conditions in multiline-expression `if` and `guard` statements align visually and have thus different indentation than both the `guard`/`if` and the body of the block.